ogr/services/pagure/service.py

```diff
@@ -257,14 +262,22 @@ def get_raw_request(
             ValueError, if JSON cannot be retrieved.
         """
 
+        updated_header = (
+            {**self.header, **header} if header is not None else self.header
+        )
+
         response = self.session.request(
             method=method,
             url=url,
             params=params,
-            headers=header or self.header,
+            headers=updated_header,
             data=data,
             verify=not self.insecure,
         )
+        headers_to_log = updated_header.copy()
+        if "Authorization" in headers_to_log:
+            headers_to_log["Authorization"] = "<redacted>"
+        logger.debug(f"Ogr sent request with following headers: {headers_to_log}")
```

Comment:
Wouldn't this be enough?
```suggestion
        logger.debug(f"Ogr sent request with following headers: {updated_header | {'Authorization': '<redacted>'}}")
```